packages/formstr-app/src/components/CopyButton/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { CheckCircleOutlined, CopyOutlined } from "@ant-design/icons";
+import { Button } from "antd";
+import { useState } from "react";
+import CopyStyle from "./copy.style";
+
+export const CopyButton = ({ getText }: { getText: () => string }) => {
+  const [copied, setCopied] = useState(false);
+  const copyText = () => {
+    navigator.clipboard.writeText(getText());
```

Comment:
This returns a promise. Should send the message that text is copied after successful promise

Promise can fail as well if user denies permission for copying. So we should then show a proper error message